lib/ckb/wallet.rb

```diff
@@ -11,9 +11,11 @@ class Wallet
     attr_reader :addr
     attr_reader :address
 
+    attr_accessor :skip_data_and_type
+
     # @param api [CKB::API]
     # @param key [CKB::Key | String] Key or pubkey
-    def initialize(api, key)
+    def initialize(api, key, skip_data_and_type: true)
```

Comment:
Why not take `skip_data_and_type ` as a parameter of `get_unspent_cells `?

Reply:
If as a parameter, you must pass this param between many methods .....

Reply:
why? `skip_data_and_type ` has a default value.